components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/OAuthAdminServiceImpl.java

```diff
@@ -1295,6 +1308,45 @@ void logOnInvalidConfig(String appName, String tokenType, long defaultValue) {
         return requestedScopeValidators;
     }
 
+
+    /**
+     * Get the IdToken Encryption Method registered by the user and filter the allowed one.
+     *
+     * @param application Application user have registered
+     * @return idTokenEncryptionMethod
+     * @throws IdentityOAuthAdminException Identity OAuthAdmin exception.
+     */
+    String filterIdTokenEncryptionMethod(OAuthConsumerAppDTO application) throws IdentityOAuthAdminException {
```

Comment:
Should this be a private method?

Reply:
Fixed with https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1470